.gitignore

```diff
@@ -5,3 +5,5 @@
 /.settings/
 /logs/
 .DS_Store
+.idea
+*.iml
```

Comment:
probably I shouldn't do it here but anyway. These are files added by IntelliJ IDEA that are to be ignored